x-pack/plugins/security_solution/public/notes/store/notes.slice.ts

```diff
@@ -318,6 +318,18 @@ export const selectNotesBySavedObjectId = createSelector(
     savedObjectId.length > 0 ? notes.filter((note) => note.timelineId === savedObjectId) : []
 );
 
+export const selectDocumentNotesBySavedObjectId = createSelector(
```

Comment:
you added a unit test to check the selector returning no notes, could you add a second one to verify we're retrieving the correct notes?